maint/code-cleanup.sh

```diff
@@ -131,18 +132,17 @@ if [ "$got_file" = "1" -a "$all" = "1" ]; then
 fi
 
 if [ "$recursive" = "1" ]; then
-    for i in `git ls-files | egrep '(\.c$|\.h$|\.c\.in$|\.h\.in$|\.cpp$|\.cpp.in$)' | \
-	egrep -v "($ignore_list)"` ; do
-	${debug} indent_code $i
+    for i in `git ls-files | egrep "($filetype_list)" | egrep -v "($ignore_list)"` ; do
 	${debug} indent_code $i
     done
```

Comment:
@wesbland Do you know why we previously double the `indent_code` line?

Reply:
I couldn’t figure it out. It didn’t make any sense to me, but we can revert this if you want.

Reply:
Checked `git blame`: one from Halim and another from Pavan, I am sure it was a result of unclean conflict resolution.

Reply:
The double `indent` was added because `indent` would sometimes toggle code layout.  Run it once, it'll add some formatting and run it a second time, it'll remove that formatting.  So, the code will always fail the white space check if we don't call it twice.

Reply:
I see. Another reason to dislike `indent` :)

Reply:
Is this something you've seen recently? We've been using this script with a pre-commit hook for a couple of years now and I haven't seen that problem. Maybe it was with an old version of `indent`?